ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx

```diff
@@ -101,48 +106,28 @@ export class WorkflowsToolbar extends React.Component<WorkflowsToolbarProps, {}>
         return Promise.all(promises);
     }
 
-    private renderActions(ctx: ContextApis): JSX.Element[] {
-        const actionButtons = [];
-        const actions: any = Actions.WorkflowOperationsMap;
-        const disabled = this.props.isDisabled;
-        const groupActions: WorkflowGroupAction[] = Object.keys(actions).map((actionName: WorkflowOperationName) => {
-            const action = actions[actionName];
-            return {
-                title: action.title,
-                iconClassName: action.iconClassName,
-                groupIsDisabled: disabled[actionName],
-                action,
-                groupAction: async () => {
-                    const confirmed = await this.performActionOnSelectedWorkflows(ctx, action.title, action.action);
-                    if (!confirmed) {
-                        return;
-                    }
-
-                    this.props.clearSelection();
-                    ctx.notifications.show({
-                        content: `Performed '${action.title}' on selected workflows.`,
-                        type: NotificationType.Success
-                    });
-                    this.props.loadWorkflows();
```

Comment:
so I actually don't think we should remove `loadWorkflows` at this point if we plan to replace it with an optimistic update per #12008. this specific call would be replaced with another call that takes the action as an argument

it would make the future change to an optimistic update more straightforward if we left this in